imports/ui/components/journalPage/Page.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Entry from './Entry';
+import ButtonNewPage from './ButtonNewPage';
+
+class Page extends Component {
+    
+    render() {
+        if (this.props.page == "right") {
+            return (
+                <div className="grid-item right-page">
+                    <Entry header="Jun 8 | Sat"/>
+                    <ButtonNewPage />
+                </div>
+            )
+        } 
+        if (this.props.page == "left") { 
+            return (
+                <div className="grid-item left-page">
+                    <Entry header="Grocery list" />
+                    <Entry header="Snack list" />
+                </div>
+            )
+        }
+    }
```

Comment:
We need to return something by default...like what if this.props doesn't contain page or page's value !== left/right ? The app will crash =/